src/ol_infrastructure/applications/odl_video_service/Pulumi.applications.odl_video_service.QA.yaml

```diff
@@ -0,0 +1,28 @@
+---
+secretsprovider: awskms://alias/infrastructure-secrets-qa
+encryptedkey: AQICAHi7xhTkB8tf1ObyPMxDhODJja4Mn4jyIo32zZVlOiZPFgFh6HjKXQ0TnfnuSPxl6JL3AAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM+0J7K+9drt+g+z83AgEQgDu10nkIqubh0RaWgj8PXlpM8P8LMoYVtNqAiWLepxKWYKhyVCaJeeZKZq6YhxZmWmsu1lMJgRYTyS50+g==
+config:
+  consul:address: https://consul-apps-qa.odl.mit.edu
+  ovs:auto_scale:
+    desired: 2
+    max: 3
+    min: 1
+  ovs:default_domain: video-rc.odl.mit.edu
+  ovs:domains:
+  - video-rc.odl.mit.edu
+  - techtv-dev.mit.edu;
+  - ttv-dev.mit.edu;
+  ovs:edx_base_url: https://courses-rc.xpro/mit.edu
+  ovs:log_level: INFO
+  ovs:rds_password:
+    secure: v1:/n8kQRfM6Od/xUKw:wwUBWVPgtmgRrN9jOA5djHdkizHjPYsUI9UDZFf5eYeOGUjZXTqSYStSK8lizXRyujf5SLfKiWqXYUeQP97QumhPuTuNbDyw+6ByeKT49P8=
+  ovs:s3_bucket_name: odl-video-service-rc
+  ovs:s3_subtitle_bucket_name: odl-video-service-subtitles-rc
+  ovs:s3_thumbnail_bucket_name: odl-video-service-thumbnail-rc
+  ovs:s3_transcode_bucket_name: odl-video-service-transcode-rc
+  ovs:s3_watch_bucket_name: odl-video-service-watch-rc
```

Comment:
It might be good to just create the buckets in Pulumi and force a bucket name for consistency and then migrate contents to the intended buckets using `s3 sync`. That way we can ensure that they are set up properly and keep them in line with what the code says.

Reply:
I like the sound of this but I think there is a complication. I am fairly sure that the bucket name is recorded in the database somewhere. I say this because at one point during my testing I had the configuration using an invalid bucket name, 'uploaded' a file, fixed the configuration, went to delete the file and the logs included the old bucket name, that theoretically the app shouldn't have known about any longer if its only reference for bucket names was the configuration items. 

That experience makes me think it is going to be more than just a s3 sync and it is going to require some analysis and SQL development.